tests/helpers.py

```diff
@@ -61,6 +65,45 @@ def open(self, *args, **kwargs):
         return super(CTFdTestClient, self).open(*args, **kwargs)
 
 
+class CTFtime:
+    @contextmanager
+    def set():
+        try:
+            set_config("start", FreezeTime.START)
+            set_config("end", FreezeTime.END)
+            yield
+        finally:
+            set_config("start", 0)
+            set_config("end", 0)
```

Comment:
I think a better default is None. I believe there are calls elsewhere in the tests so maybe use whatevers there.